tests/MimeUploadValidatorTest.php

```diff
@@ -1,4 +1,12 @@
 <?php
+
```

Comment:
Would you mind adding namespaces to the tests too? `SilverStripe\MimeValidator\Test` - add PSR-4 autoloader for this too to composer.json